### PR TITLE
Feat/tsemach/add integration app template

### PR DESCRIPTION
**Pull request by tsemachLi**

### CODE REVIEW

File: apps/app-templates/integration/.editorconfig
Comment (gregra81) [inline apps/app-templates/integration/.editorconfig:1]: do we need this file to make the sample work?

File: apps/app-templates/integration/.eslintrc.json
Comment (gregra81) [inline apps/app-templates/integration/.eslintrc.json:1]: do we need this file to make the sample work?

File: apps/app-templates/integration/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Dor Shaked
```
Comment (gregra81) [inline apps/app-templates/integration/LICENSE:3]: let's remove here Dor Shaked

File: apps/app-templates/integration/package.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run server",
+    "dev": "npm run dev:stop && concurrently \"npm run dev:server\" \"npm run dev:expose\"",
+    "dev:server": "nodemon ./src/app.ts",
+    "dev:expose": "mapps tunnel:create -p 8080",
+    "dev:stop": "kill-port 8080",
+    "server": "NODE_ENV=production node ./src/app.ts",
+    "deploy": "npm run mapps-updated code:push",
+    "logs": "npm run mapps-updated code:logs",
```
Comment (gregra81) [inline apps/app-templates/integration/package.json:14]: let's not put monday code here

File: apps/app-templates/integration/package.json [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  "dependencies": {
+    "@mondaycom/apps-sdk": "^0.1.3",
+    "@octokit/rest": "^16.43.2",
+    "body-parser": "^1.20.2",
+    "dotenv": "^16.0.3",
+    "eslint-config-prettier": "^9.1.0",
+    "express": "^4.17.1",
+    "jsonwebtoken": "^9.0.0",
+    "monday-sdk-js": "^0.1.4",
+    "prettier": "^3.2.4"
```
Comment (gregra81) [inline apps/app-templates/integration/package.json:34]: move to dev dep

**Review by gregra81 — COMMENTED** [review]
good start, let's discuss it online

File: apps/app-templates/integration/package.json [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    "start": "npm run server",
+    "dev": "npm run dev:stop && concurrently \"npm run dev:server\" \"npm run dev:expose\"",
+    "dev:server": "nodemon ./src/app.ts",
+    "dev:expose": "mapps tunnel:create -p 8080",
+    "dev:stop": "kill-port 8080",
+    "server": "NODE_ENV=production node ./src/app.ts",
+    "deploy": "npm run mapps-updated code:push",
+    "logs": "npm run mapps-updated code:logs",
+    "mapps:init": "npm run mapps-updated init",
+    "mapps-updated": "mapps -u",
```
Comment (tsemachLi) [inline apps/app-templates/integration/package.json:16]: remove

File: apps/app-templates/integration/package.json (hunk 2 above)
Comment (tsemachLi) [inline apps/app-templates/integration/package.json:14]: remove

File: apps/app-templates/integration/package.json [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    "@types/jsonwebtoken": "^9.0.5",
+    "@types/node": "^20.11.10",
+    "@typescript-eslint/eslint-plugin": "^6.19.1",
+    "@typescript-eslint/parser": "^6.19.1",
+    "concurrently": "^8.2.2",
+    "eslint": "^8.56.0",
+    "eslint-config-standard": "^17.1.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-n": "^16.6.2",
+    "eslint-plugin-promise": "^6.1.1",
```
Comment (tsemachLi) [inline apps/app-templates/integration/package.json:50]: remove

File: apps/quickstart-fullstack-react-node/README.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+## Run the project
+
+1. Add your MONDAY_SIGNING_SECRET to .env file
+   <br> \*\* To get your MONDAY_SIGNING_SECRET go to monday.com, open Developers section, open your app and find the Signing Secret in "Basic Information" section
+   <br> ![Screenshot](https://dapulse-res.cloudinary.com/image/upload/f_auto,q_auto/remote_mondaycom_static/uploads/VladMystetskyi/4db4f03e-67a5-482d-893e-033db67ee09b_monday-Apps2020-05-1901-31-26.png)
+2. Build the client code and run the server with ngrok tunnel with the command:
+
+```bash
+yarn dev
```
Comment (gregra81) [inline apps/quickstart-fullstack-react-node/README.md:28]: change to what is the final command

File: apps/quickstart-fullstack-react-node/README.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+## Deploying to [monday code](https://developer.monday.com/apps/docs/hosting-your-app-with-monday-code) hosting
+
+
+1. For the first-time deployment, use this command to initialize Monday code:
+
+2. Start deployment to monday-code using the following command [monday apps cli](https://github.com/mondaycom/monday-code-cli#mapps-codepush):
+
+
+```bash
+yarn deploy
```
Comment (gregra81) [inline apps/quickstart-fullstack-react-node/README.md:44]: change to what is the final command

File: apps/quickstart-fullstack-react-node/client/README.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+## Overview
+This is the "Quickstart React" example Monday app. 
+<br>It can be used as a board view, item view or any other view, connected to a board and render data from the board using settings.
+
+<br>This app demonstrates how to use: 
+- [settings](https://github.com/mondaycom/monday-sdk-js#mondaygettype-params--) 
+- [context](https://github.com/mondaycom/monday-sdk-js#mondaygettype-params--) 
+- [API](https://github.com/mondaycom/monday-sdk-js#mondayapiquery-options--)
```
Comment (gregra81) [inline apps/quickstart-fullstack-react-node/client/README.md:8]: these links don't work

File: apps/quickstart-fullstack-react-node/client/README.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+<br>You can find more info in our QuickStart guide [here](https://monday.com/developers/apps/quickstart-view/)
```
Comment (gregra81) [inline apps/quickstart-fullstack-react-node/client/README.md:10]: let's link to this repo, right?

File: apps/quickstart-fullstack-react-node/client/package.json [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+    "start": "npm run stop && concurrently \"npm run server\" \"npm run expose\"",
+    "build": "react-scripts build",
+    "expose": "mapps tunnel:create -p 8301",
+    "server": "react-scripts start",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "stop": "kill-port 8301 && kill-port 4049 && kill-port 4040"
+  },
+  "dependencies": {
+    "@mondaycom/apps-sdk": "^2.1.2",
```
Comment (gregra81) [inline apps/quickstart-fullstack-react-node/client/package.json:15]: why do we need it?

File: apps/quickstart-fullstack-react-node/yarn.lock
Comment (gregra81) [inline apps/quickstart-fullstack-react-node/yarn.lock:1]: remove - should use package-lock.json

**Review by gregra81 — APPROVED** [review]
Great! 🚀

**Pull request merged**